drivers/pwm/pwm_nrfx.c

```diff
@@ -339,10 +340,16 @@ static int pwm_nrfx_pm_action(const struct device *dev,
 
 #define PWM(dev_idx) DT_NODELABEL(pwm##dev_idx)
 #define PWM_PROP(dev_idx, prop) DT_PROP(PWM(dev_idx), prop)
+#define PWM_HAS_PROP(idx, prop)	DT_NODE_HAS_PROP(PWM(idx), prop)
 
 #define PWM_NRFX_DEVICE(idx)						      \
 	NRF_DT_CHECK_NODE_HAS_PINCTRL_SLEEP(PWM(idx));			      \
-	static struct pwm_nrfx_data pwm_nrfx_##idx##_data;		      \
+	static uint16_t pwm_##idx##_seq_values			       \
+			[NRF_PWM_CHANNEL_COUNT]		       \
+			PWM_MEMORY_SECTION(idx);			       \
+	static struct pwm_nrfx_data pwm_nrfx_##idx##_data = { \
+		.seq_values = pwm_##idx##_seq_values, 		      \
```

Comment:
The `seq_values` pointer could be actually moved to `struct pwm_nrfx_config`. This initialization wouldn't be needed then. And a few bytes would be saved.

Reply:
done